qa-catalogue

```diff
@@ -78,24 +79,32 @@ SCHEMA:                   ${SCHEMA:-}
 TYPE_PARAMS (params):     ${TYPE_PARAMS:-}
 UPDATE:                   ${UPDATE:-}
 VERSION:                  ${VERSION:-}
-RUN_WEB_CONFIG:           ${RUN_WEB_CONFIG}
+WEB_CONFIG:               ${WEB_CONFIG}
 ---------------------------
 EOF
 
+# don't proceed if web configuration shall be updated but cannot do so
+if [[ ! -z "$WEB_CONFIG" ]]; then
+  if [[ ! -f "$WEB_CONFIG" ]]; then
+    echo "web configuration file missing: $WEB_CONFIG" >&2
+    exit 1
+  elif [[ -z "$CATALOGUE" ]]; then
+    echo "catalogue short name not set" >&2
+    exit 1
+ fi
+fi
+
+# run actual analysis
 . ./common-script ${1:-}
 
-if [[ ${RUN_WEB_CONFIG} = 1 ]]; then
-  if [[ "${CATALOGUE:-}" != "" ]]; then
-    FILE=/var/www/html/qa-catalogue/configuration.cnf
-    if [[ -f $FILE ]]; then
-      HAS_CATALOGUE=$(grep -c 'catalogue=' $FILE || true)
-      if [[ $HAS_CATALOGUE == 1 ]]; then
-        grep -vP "^catalogue=" $FILE > /tmp/catalogue
-        mv /tmp/catalogue $FILE
-      fi
-      echo "catalogue=$CATALOGUE" >> $FILE
-    fi
+# optionally update web configuration
+if [[ ! -z "$WEB_CONFIG" ]]; then
+  HAS_CATALOGUE=$(grep -cP '^\*scatalogue\s*=' $WEB_CONFIG || true)
```

Comment:
there is a typo here: `^\*scatalogue\s*='`. Could you please change `\*s` to `\s*`?

Reply:
Thanks for spotting. I've fixed it!